### ISSUE TITLE
Details of pre-training with dVAE tokens

### ISSUE THREAD

**Issue by netw0rkf10w** [facebookresearch/mae#115]
Dear MAE's authors,

Thank you so much for having made the code and the pre-trained models available. 

The paper indicates that dVAE achieved very good results (on par with MAE), without providing much details on the pre-training settings:

<img width="652" alt="mae_dvae" src="https://user-images.githubusercontent.com/8569515/184371632-696b23ce-1862-4ae9-8b6b-dade8c486c1b.png">

Therefore, I would like to ask for more details of pre-training with dVAE. More specifically:

1. In addition to the discrete tokens, did you feed the image patches to the model as well (as done in BEiT)?
2. What was the masking ratio used for dVAE and did you tune this value? Without any indication from the paper, I understood that you used the same ratio as MAE, which is 75%. 

Thank you very much in advance for your answers!
